docs/guides/validators/security_best_practices.md

```diff
@@ -373,25 +378,30 @@ You can find used ports in your node config files and make your own firewall rul
 ~/.haqqd/config/client.toml
 ```
 
-Here is the list of used services (by default):
+**Here is the best practice of used services:**
 
-- Address defines the API server to listen on `tcp://0.0.0.0:1317`
-- Rosseta API `:8080`
-- gRPC server `0.0.0.0:9090`
-- gRPC web `0.0.0.0:9091`
-- EMV RPC HTTP `0.0.0.0:8545`
-- EMV WebSocket `0.0.0.0:8546`
-- Tendermint RPC `tcp://localhost:26657`
-- ABCI application `tcp://127.0.0.1:26658`
-- pprof (Go pkg) `localhost:6060`
-- p2p incoming connections `tcp://localhost:26656`
-- Prometheus `:26660`
+| port | service | used by | rpc node | validator |
+|---|---|---|---|---|
+| 1317 | API server |  | ✅ | ❌ 
+| 8080 | Rosseta API |  | ❌ | ❌ |
+| 9090 | gRPC server |  | ✅ | ❌ |
+| 1317 | gRPC web |  | ✅ | ❌ |
+| 8545 | EMV RPC HTTP |  | ✅ | ❌ |
+| 8546 | EMV RPC WebSocket |  | ✅ | ❌ |
+| **26657** * | Tendermint RPC |  | ✅ | ❌ |
+| 26658 | ABCI application |  | ❌ | ❌ |
+| 6060 | pprof (Go pkg) |  | ❌ | ❌ |
+| 26656 | p2p incoming connections |  | ✅ | ✅ |
+| 26660 | Prometheus |  | ❌ | ❌ |
 
-::: tip
+<br>
+
+:::danger
 
-For security reasons, we recommend that you use only the services and ports necessary for your node to work.
+* Port `26657` is used for working with node CLI. For security reasons we recommended to close this port access from public network.
```

Comment:
it's a bit confusing to note this only for tendermint rpc but not for other RPCs.

Reply:
Changed with text about all listed ports in table